pkgs/by-name/or/ory/package.nix

```diff
@@ -3,32 +3,38 @@
   buildGoModule,
   fetchFromGitHub,
   installShellFiles,
+  nix-update-script,
+  versionCheckHook,
 }:
 
-buildGoModule rec {
+buildGoModule (finalAttrs: {
   pname = "ory";
-  version = "0.3.4";
+  version = "v1.1.0";
 
   src = fetchFromGitHub {
     owner = "ory";
     repo = "cli";
-    tag = "v${version}";
-    hash = "sha256-q7+Fpttgx62GbKxCCiEDlX//e/pNO24e7KhhBeGRDH0=";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-+Su2FIuCb2vpPW/OCOTzqQOZPpY9gGRbwylSepLh2hk=";
```

Comment:
you have to reset the hash every time you make a change to the fetcher inputs to test it